packages/monorepo-scripts/src/utils/github_release_utils.ts

```diff
@@ -12,7 +12,10 @@ import { utils } from './utils';
 
 const publishReleaseAsync = promisify(publishRelease);
 // tslint:disable-next-line:completed-docs
-export async function publishReleaseNotesAsync(packagesToPublish: Package[], isDryRun: boolean): Promise<void> {
+export async function publishReleaseNotesAsync(
+    packagesToPublish: Package[],
+    isDryRun: boolean,
+): Promise<string | undefined> {
```

Comment:
Looks like we are returning `aggregateNotes` which is initialized as an empty string, is it necessary to expand the return type here to cover `undefined`?

Reply:
See line 76, if `isDryRun` there's a blank `return;` which returns undefined